openedx/core/djangoapps/zendesk_proxy/tests/test_views.py

```diff
@@ -0,0 +1,120 @@
+"""Tests for zendesk_proxy views."""
+from copy import deepcopy
+import ddt
+import json
+from mock import MagicMock, patch
+
+from django.core.urlresolvers import reverse
+from django.test.utils import override_settings
+
+from openedx.core.lib.api.test_utils import ApiTestCase
+from openedx.core.djangoapps.zendesk_proxy.v0.views import ZENDESK_REQUESTS_PER_HOUR
+
+
+@ddt.ddt
+@override_settings(
+    ZENDESK_URL="https://www.superrealurlsthataredefinitelynotfake.com",
+    ZENDESK_OAUTH_ACCESS_TOKEN="abcdefghijklmnopqrstuvwxyz1234567890"
+)
+class ZendeskProxyTestCase(ApiTestCase):
+    """Tests for zendesk_proxy views."""
+
+    def setUp(self):
+        self.url = reverse('zendesk_proxy_v0')
+        self.request_data = {
+            'name': 'John Q. Student',
+            'tags': ['python_unit_test'],
+            'email': {
+                'from': 'JohnQStudent@example.com',
+                'subject': 'Python Unit Test Help Request',
+                'message': "Help! I'm trapped in a unit test factory and I can't get out!",
+            }
+        }
+        return super(ZendeskProxyTestCase, self).setUp()
+
+    def test_post(self):
+        with patch('requests.post', return_value=MagicMock(status_code=201)) as mock_post:
+            response = self.request_with_auth(
```

Comment:
I guess these should be publicly accessible, so it would be more like, `request_without_auth`?